products/cpdb/bash/05_export.sh

```diff
@@ -18,6 +18,8 @@ mkdir -p output && (
     csv_export geospatial_check &
     csv_export source_data_versions &
```

Comment:
Do we care about consistency between

```bash
csv_export source_data_versions
```

vs

```bash
cp ../source_data_versions.csv ./
```

Both are perfectly valid in the current setup - `load.py` creates the table, then loads it into the build engine

Reply:
personally prefer `csv_export`. feels like that keeps it explicit that `source_data_versions` is one of a many exports and if we ever want them all to go to a new directory we'd only have to change that in one place